data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+					goto snowdig
+			label back
+			`	She helps you up, and you continue on your way down the hills. You get back to the forest, and she starts finding the trail back to the ship. Halfway through, the storm gets progressively weaker.`
+			`	When you two return to your ship, you leave to pick up Kirrie and Tugroob, who also come back empty handed.`
+				accept
+			label snowdig
+			`	You start clawing your way down the snow as Iplora asks you to stop and continue moving toward the ship. You continue, your hands getting colder as the snow steals the heat even through your gloves. You hit a hard surface and start clearing from there, only to see it was, in fact, just a boulder.`
+			`	"Satisfied? Return, we must. Come now," You turn to see her offering her hand once again.`
+			choice
+				`	(Take her hand.)`
+					goto back
+				`	(Head to another mound and try digging there.)`
+			label keepdig
+			`	You stand up and run to another spot where you fell over. Iplora follows you, and even with the snow muffling the sounds around you, you can hear her exasperated sigh. She tries to pull you away, but you hit a hard surface yet again, this time creating a faint metallic echo. She takes her hands off of you and lays down, now helping you dig out whatever is there. Once the crate is visible, she stands up. It's large and looks like a coffin, but much of it is still buried. She asks you to help her get some hooks on it, after which she struggles to try and pull it out.`
+			`	The slippery snow certainly doesn't help her hooves, but it does make the crate slippery as well. It's coaxed out of the ground, bit by bit. You help her as best as you can and eventually get it on top of the snow field. Iplora searches through some tools in her bag and takes out what looks like a crowbar. She starts opening it, a dim light creeping out, but an energy blast coming from the the top of the hills hits the ground near the hole. It misses you by just about a meter.`
+			`	She yells for you to get down, pulling out her weapon and firing in the direction the shot came from.`
+			apply
+				"resistance evidence" ++
+			choice
+				`	(Take cover.)`
+				`	(Pull out your gun and help her.)`
+					goto help
+			`	You get inside the hole, hoping you'll have enough protection from the blasts. You can still hear Iplora move around, and you occasionally peek your head over to see her shooting and taking cover, running and dodging incoming blasts. The shots stop and you see her come over, saying that it's safe to get out now.`
+			`	"Fled, our attacker has," she claims. From the looks of it she wasn't injured. You two get back to the crate.`
+				goto opencrate
+			label help
+			`	You start shooting the hills above, trying to aim around where Iplora's own shots go to. A shot comes your way, and you jump to the side. Iplora takes cover inside the hole, firing a few more shots from there, then leaves up the hill. You follow her lead, approaching from another direction.`
```

Comment:
How big is this hole? She's a centaur for pity's sake - with hooves that actually let her manoeuvre somewhat well. Also; see deathtrap.

Reply:
Big-ish?
Centaur or not, dodging shots like that implies more superpowers than good maneuverability.

I'll add some more cover to the snowfield that she could realistically go to.